src/common/components/ConnectWalletButton/ConnectWalletButton.tsx

```diff
@@ -5,7 +5,9 @@ type ConnectWalletButtonProps = ButtonProps;
 export const ConnectWalletButton = ({ ...props }: ConnectWalletButtonProps) => {
   return (
     <Button
+      as="a"
       fontSize={{ sm: 'sm', md: 'md' }}
+      href="/sign-in"
```

Comment:
Do you think it’ll be better if we wrap this and use next/link instead for linking to a page?